pkg/cmd/opts/kustomize_test.go

```diff
@@ -0,0 +1,26 @@
+package opts
+
+import (
+	"fmt"
+	"os/exec"
+	"testing"
+
+	"github.com/jenkins-x/jx/pkg/cmd/clients"
+)
+
+// TestCommonOptions_EnsureKustomize tests that Kustomize is
+//  properly installed
+func TestCommonOptions_EnsureKustomize(t *testing.T) {
+	cmd := exec.Command("kustomize", "version")
```

Comment:
This is bit tricky, since it means depending on your local state the tests might do something or not. Really, you want `o.EnsureKustomize()` always to be run and it should always succeed.

You also want to make sure that your test is not interfering with your local JX_HOME which you use for development. 

So what you can do is:

- make sure a temporary JX_HOME is used - https://github.com/jenkins-x/jx/blob/23f9f49350a8aba8bab8ceaf1ff4a1fee8b288ed/pkg/cmd/testhelpers/cmd_test_helpers.go#L149
- make sure the temporary directory is deleted - https://github.com/jenkins-x/jx/blob/23f9f49350a8aba8bab8ceaf1ff4a1fee8b288ed/pkg/cmd/testhelpers/cmd_test_helpers.go#L186
- set o.NoBrew to _true_ to force an install w/o Brew, but via a download from GitHub. In this case the binary ends up in JX_HOME/bin/ (one can then also check via the file system whether the install worked)